srcpkgs/libdeflate/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'libdeflate'
+pkgname=libdeflate
+version=1.8
+revision=1
+build_style=gnu-makefile
+make_build_args="PREFIX=/usr PROG_SUFFIX=libdeflate"
```

Comment:
PREFIX should not be required..? https://github.com/ebiggers/libdeflate/blob/master/Makefile#L80

Reply:
Yep, fixed. Thanks :)